### ISSUE TITLE
[BUG] itemSize: large, is not set as default

### ISSUE THREAD

**Issue by polhaghverdian** [Lissy93/dashy#576]
### Environment

Self-Hosted (Docker)

### Describe the problem

Hi,

I updated from 1.9.0 to 2.0.5,  and dashy doesn't seem to apply my pre-defined "itemSize", instead I get medium as the default.

edit: I also tried version 2.0.0, same problem.

Example snippet below from my conf.yml:

```
sections:
- name: Management
  displayData:
    itemSize: large
  items:
    - title: Cockpit
      description: A sysadmin login session in a web browser
      icon: "cockpit.png"
      url: https://192.168.2.77
      statusCheck: true
      statusCheckAllowInsecure: true
      statusCheckHeaders:
         Accept: text/html
```

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Hey @polhaghverdian 
Sorry about that, seems that property slipped my mind when I was migrating to VueX. Fixed in #557 / 2.0.6 
Feel free to reach out if you need anything else :)

**Comment by polhaghverdian** [comment]
@Lissy93 Thank you so much! :)
